src/web/app/pages-instructor/instructor-student-list-page/instructor-student-list-page.component.html

```diff
@@ -1,157 +1,47 @@
-              </ng-container>
-            </ng-container>
+          <div class="col-sm-1">
+            <button *ngIf="!viewStudent[idx]"
+              class="btn float-right"><i class="fas fa-chevron-circle-down fa-lg"
+              style="color: white;"></i></button>
+            <button *ngIf="viewStudent[idx]"
+              class="btn float-right"><i class="fas fa-chevron-circle-up fa-lg" style="color: white;"></i></button>
           </div>
-
-          <div class="col-sm-3 float-left">
-            <ng-container *ngIf="courseDetailsList.length">
-              <div class="text-color-primary pb-3">
-                <strong> Students </strong>
-              </div>
-              <div class="checkbox-outer-div pb-3" [hidden]="!isAnyPresentCoursesChecked">
-                <input type="checkbox"
-                       (change)="toggleAllPresentStudentDetailsState(!isAllPresentStudentDetailsChecked)"
-                       [checked]="isAllPresentStudentDetailsChecked">
-                <label class="ml-1"><strong> Show Emails </strong></label>
-              </div>
-              <ng-container *ngFor="let courseDetails of courseDetailsList">
-                <ng-container *ngFor="let section of courseDetails.sections">
-                  <ng-container *ngFor="let team of section.teams">
-                    <div class="checkbox-outer-div" *ngFor="let student of team.students" [hidden]="!team.isChecked">
-                      <input type="checkbox" [(ngModel)]="student.isChecked">
-                      <label class="ml-1">
-                        {{student.email}}
-                      </label>
-                    </div>
-                  </ng-container>
-                </ng-container>
-              </ng-container>
-            </ng-container>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-</div>
-
-<ng-container *ngIf="courses && courses.length" >
-  <div class="p-2">
-    <div *ngFor="let course of courses" class="card border mb-3 p-0"
-         [hidden]="(!course.isChecked) && (!isDisplayArchive) && course.isArchived">
-      <div class="card-header font-weight-bold"
-           [ngClass]="{'text-dark-blue bg-light-blue ': !course.isArchived, 'bg-light-grey': course.isArchived}"
-           (click)="toggleCourseStateAtInput(course)">
-        <strong>[{{course.id}}]:</strong> {{course.name}}
-        <div class="arrow-sign">
-          <button class="btn btn-info btn-xs"
-                  routerLink="/web/instructor/courses/enroll"
-                  [queryParams]="{courseid: course.id}">
-            Enroll
-          </button>
-          <i class="fas fa-chevron-up pl-3" [hidden]="!course.isChecked"></i>
-          <i class="fas fa-chevron-down pl-3" [hidden]="course.isChecked"></i>
         </div>
       </div>
-      <div class="card-body p-0">
+      <div class="card-body" *ngIf="viewStudent[idx]">
+        <button class="btn" style="background-color: #127c2b; color: white;"
```

Comment:
After trying it out, I'd suggest against hardcoding the button style (versus using Bootstrap's default settings). In places that use Bootstrap buttons, when I hover over them, there's feedback to the user (button color gets darker), but not with this button.

Reply:
Yes that is a good point. Let us stick with bootstrap colours for now then. (You could update the css to have a darker/lighter shade on hover if needed but thats pretty unnecessary)